src/System.IO.Packaging/src/System/IO/Packaging/InternalRelationshipCollection.cs

```diff
@@ -354,7 +360,7 @@ private void ProcessRelationshipAttributes(XmlCompatibilityReader reader)
             if (string.IsNullOrEmpty(targetAttributeValue))
                 throw new XmlException(SR.Format(SR.RequiredRelationshipAttributeMissing, s_targetAttributeName), null, reader.LineNumber, reader.LinePosition);
 
-            Uri targetUri = new Uri(targetAttributeValue, UriKind.RelativeOrAbsolute);
+            Uri targetUri = new Uri(targetAttributeValue, DotNetRelativeOrAbsolute);
```

Comment:
`Uri targetUri = new Uri(targetAttributeValue, targetMode == TargetMode.Internal ?
 UriKind.Relative : UriKind.RelativeOrAbsolute);`
would make this work on mono too, right?

Reply:
Yes, although that will start throwing if it happened to be an absolute path at this location rather later on.